index.html

```diff
@@ -5171,6 +5219,11 @@ <h2 id="AudioWorklet-Examples">
       this.nextUpdateFrame += this.interval;
       this.postMessage({ volume: this.volume });
     }
+
+    // Retain an active reference if the volume is above a threshold,
+    // so that disconnecting inputs does not immediately cause the meter
+    // to stop computing its smoothed value.
+    return this.volume &gt;= VU_METER_MINIMUM;
```

Comment:
This is a bit of a forced example as a VU meter doesn't produce output, and so anything consuming its `volume` value would retain a reference to the node anyway.

A good additional example would be a source node with no inputs that returns `true` while the source is active.

Another good example would be a node emitting an output with a true tail-time based on the input. It would note the time interval of any continuous sequence of processing batches in which `contextInfo.connected` was true, and if this interval reached the tail time, would return `false` from `process()`.